sync/src/snapshot/mod.rs

```diff
@@ -16,5 +16,6 @@
 
 mod error;
 mod service;
+mod snapshot;
```

Comment:
It just modularizes the component you created in the same PR.
Why didn't you create a new module at the beginning?